index.html

```diff
@@ -0,0 +1,133 @@
+            <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+            <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+        </ul>
+        <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+        <div class="platform">
+            <button class="pla__btn">html</button>
+            <button class="pla__btn">css</button>
+            <button class="pla__btn">javascript</button>
+        </div>
+        <button class="see-project">See Project</button>
+    </section>
+
+    <section class="about-section">
+        <div class="details-about-me">
+            <h1 class="bold-text">About Myself</h1>
+            <p class="signup-disc">i am a software developer by profesion and i help develop many softwares, you can go ahed and check out some of my projects in my portofolio, both mobile and web based and continuing to do so. if you need help regarding software development or have a project in mind feel free to contact me.</p>
+            <h2 class="let-connect">LET'S CONNECT</h2>
+        </div>
+        <ul class="contact_us_wrapper">
+            <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
+            <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+            <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+            <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>  
+        </ul>
+        <button class="see-project">See Project</button>
+        <div class="performance">
+            <h1 class="languages-title">Languages</h1>
+            <div class="language-details">
+                <h2 class="javascript"><img src="./img/javascript.png" alt="javscript picture">Javascript</h2>
+                <h2 class="html"><img src="./img/html.png" alt="html picture">HTML</h2>
+                <h2 class="css"><img src="./img/css.png" alt="css picture">CSS</h2>
+            </div>
+            <div class="total__wrapper">
+                <div class="platform__wrapper">
+                    <h1 class="frameworks-title">Frameworks</h1>
+                    <div class="bar__wrapper">
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </div>
+                </div>
+                <div class="underline"></div>
+                <div class="platform__wrapper">
+                    <h1 class="frameworks-title">Skills</h1>
+                    <div class="bar__wrapper">
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </div>
```

Comment:
- [ ]  Please use list items for Languages, Frameworks and Skills as per the requirements.